data/README.md

```diff
@@ -0,0 +1,68 @@
+# The `data` Directory
+
+## Overview
+
+This directory contains files used for pre-loading our production server
+database with *Pseudomonas aeruginosa*-related samples and annotations. All of
+these files are loaded into the system using the management commands listed in
+[the load_default_pseudomonas_data.sh script](https://github.com/greenelab/adage-server/blob/master/load_default_pseudomonas_data.sh).
+These commands take the location and filename of the desired data files as
+an argument.
+
+If using [fabric](http://www.fabfile.org/), these can also be loaded using the
+`import_data_and_index()` function in
+[fabfile/adage_server.py](https://github.com/greenelab/adage-server/blob/master/fabfile/adage_server.py).
+The
+[adage/adage/config.py.template file](https://github.com/greenelab/adage-server/blob/master/adage/adage/config.py.template)
+specifies which files are loaded by this `import_data_and_index()` function.
+
+All of the files below are tab-delimited text files (the decompressed form
+of the gzipped file `gene_gene_network_cutoff_0.2.txt.gz` is a tab-delimited
+text file as well). If you wish to load your own files into your adage-server
+instance, they should be in the same format as the files below.
+
+## File Listing and Description
+
+### **PseudomonasAnnotation.tsv**
+  Lists samples retrieved from ArrayExpress along with manually-curated
+  annotations for those samples. Each row is a sample, and the columns
+  contain information about that sample.
+
+### **all-pseudomonas-gene-normalized.pcl**
```

Comment:
This file is generated using the script `data_collection.sh` in the eADAGE bitbucket repo. Link here https://bitbucket.org/greenelab/eadage/src/d8a98586e0f448a5b7d497f5516cc4e241635361/data_collection/data_collection.sh?at=default&fileviewer=file-view-default